packages/node/package.json

```diff
@@ -18,7 +18,7 @@
     "test:e2e:watch": "SILENCE_LOGGER=true jest --coverage --watch --selectProjects e2e",
     "test": "SILENCE_LOGGER=true jest --coverage --selectProjects unit",
     "test:watch": "SILENCE_LOGGER=true jest --coverage --watch --selectProjects unit",
-    "test:watch:debug": "jest \"call-api.test\" --selectProjects unit --watch"
+    "test:watch:debug": "jest \"evm/handlers/initialize-provider.test.ts\" --selectProjects unit --watch"
```

Comment:
Nit: I think this command should not be part of package json, but a tip in README instead.

Reply:
I agree. It is convenient but a little bit weird that one needs to change the code everytime a single test suite needs to be run. I'll remove it and add a note on the README.